src/main/java/org/spongepowered/api/block/BlockType.java

```diff
@@ -121,4 +122,18 @@
      */
     float getEmittedLight();
 
+    /**
+     * Gets if the block is able to be gathered using the silk touch
+     * enchantment.
+     *
+     * @return If the block can be harvested with silk touch
+     */
+    boolean canHarvestWithSilk();
+
+    /**
+     * Gets the equivalent {@link ItemBlock} for this BlockType.
+     *
+     * @return The equivalent {@link ItemBlock}
+     */
+    ItemBlock getHeldItem();
```

Comment:
Blocks with no directly corresponding ItemBlock include: flowing_water, water, flowing_lava, lava, bed, piston_head, piston_extension, double_stone_slab, fire, redstone_wire, wheat, lit_redstone_ore, unlit_redstone_torch, reeds, portal, cake, unpowered_repeater, powered_repeater, pumpkin_stem, melon_stem, nether_wart, brewing_stand, cauldron, end_portal, lit_redstone_lamp, double_wooden_slab, cocoa, tripwire, flower_pot, carrots, potatoes, skull, unpowered_comparator, powered_comparator, standing_banner, wall_banner, daylight_detector_inverted, double_stone_slab2, spruce_door, birch_door, jungle_door, acacia_door, and dark_oak_door. Many of these have ItemTypes which associate with them, of course, but several simply do not have `ItemBlock` variants.